ranger/config/rc.conf

```diff
@@ -531,7 +534,7 @@ map cm search_next order=mtime
 map ca search_next order=atime
 
 # Tabs
-map <C-n>     tab_new
+map <C-n>     tab_new .
```

Comment:
This is a noop afaict?

Reply:
No; `tab_new` by default opens each tab in the home directory. Which I found quite annoying..

Reply:
That's simply false as far as I can ascertain. I'm not making this up, I tested it back then and I tested it again now. `C-n` opens a new tab in whatever directory you're in in ranger.